arcgis-ios-sdk-samples/Layers/Query a map image sublayer/QueryMapImageSublayer.storyboard

```diff
@@ -24,36 +23,50 @@
                                     <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                                     <subviews>
-                                        <textField opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" borderStyle="roundedRect" placeholder="Population 2000 &gt;" textAlignment="natural" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="Jkh-zh-w1R">
-                                            <rect key="frame" x="8" y="8" width="293" height="30"/>
-                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                                            <nil key="textColor"/>
-                                            <fontDescription key="fontDescription" type="system" pointSize="14"/>
-                                            <textInputTraits key="textInputTraits" autocorrectionType="no" spellCheckingType="no" keyboardType="numberPad" returnKeyType="done" smartDashesType="no" smartInsertDeleteType="no" smartQuotesType="no"/>
-                                            <connections>
-                                                <outlet property="delegate" destination="rtM-Ml-JFc" id="xVF-g1-7SJ"/>
-                                            </connections>
-                                        </textField>
-                                        <button opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="9Co-ma-VW1">
-                                            <rect key="frame" x="309" y="7" width="42" height="30"/>
-                                            <state key="normal" title="Query"/>
-                                            <connections>
-                                                <action selector="query:" destination="rtM-Ml-JFc" eventType="touchUpInside" id="T1j-k2-zy8"/>
-                                            </connections>
-                                        </button>
+                                        <stackView opaque="NO" contentMode="scaleToFill" spacing="8" translatesAutoresizingMaskIntoConstraints="NO" id="wbD-0d-BEI">
+                                            <rect key="frame" x="8" y="8" width="343" height="33"/>
+                                            <subviews>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="POP2000 &gt;" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="MBv-OZ-RE4">
```

Comment:
Entering a sufficiently large number will cause the label to be truncated. The horizontal content compression resistance priority of the label should be larger than that of the text field. `horizontalCompressionResistancePriority="751"` would do the trick.